R/DBI-pool.R

```diff
@@ -39,13 +39,13 @@ setMethod("onValidate", "DBIConnection", function(object) {
     options <- c(
       "SELECT 1",
       "SELECT 1 FROM DUAL",
-      "SELECT 1 FROM INFORMATION_SCHEMA.SYSTEM_USERS",
-      "SELECT * FROM INFORMATION_SCHEMA.TABLES",
+      "SELECT 1 FROM INFORMATION_SCHEMA.SYSTEM_USERS WHERE 0=1",
+      "SELECT 1 FROM INFORMATION_SCHEMA.TABLES WHERE 0=1",
       "VALUES 1",
-      "SELECT 1 FROM SYSIBM.SYSDUMMY1",
-      "select count(*) from systables",
-      "select count(*) from SYS_TABLES",
-      "select count(*) from SYS.TABLES"
+      "SELECT 1 FROM SYSIBM.SYSDUMMY1 WHERE 0=1",
```

Comment:
The additional where clause is likely unnecessary on the `SYSIBM.SYSDUMMY1` table. As noted [here](https://www.ibm.com/docs/en/db2-for-zos/12?topic=tables-sysdummy1), the table exists with only one row for when the query would otherwise fail due to a missing table reference, I.E. `SELECT 1.`

Sorry if this is not the right place to leave this comment.